curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64475c0b61cddb6feaab4e2e.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-13
 
 Now you can alert the user if they don't enter a number, or the number is invalid before you attempt to convert it into binary.
 
-In the body of the `if` statement, use the `alert()` method to display the text `Please provide a decimal number`.
+In the body of the `if` statement, use the `alert()` method to display the text `please provide a decimal number greater than or equal to 0`.
```

Comment:
```suggestion
In the body of the `if` statement, use the `alert()` method to display the text `Please provide a decimal number greater than or equal to 0`.
```

Can we bring the first word, first letter as capital please ? for all the steps

Reply:
Done